### PR TITLE
[DOCS] Elastic Security UI updates

### LINKED ISSUE #2547: [DOCS] Elastic Security UI updates
A few changes to the [security UI](https://www.elastic.co/guide/en/security/master/es-ui-overview.html): 

- New Entity Analytics dashboard https://github.com/elastic/security-docs/issues/2517
- New Intelligence section https://github.com/elastic/security-docs/issues/2497

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves #2547.

‼️ Note: Depends on #2526 being merged and published first, so that we can link from the Elastic Security UI page to the Indicators page (which doesn't exist yet in `main`). The link currently points to the Investigate landing page to keep CI from breaking.

Preview: These subsections in the Elastic Security UI topic:
   * [Dashboards](https://security-docs_2594.docs-preview.app.elstc.co/guide/en/security/master/es-ui-overview.html#_dashboards) - updated section to include Entity Analytics dashboard
   * [Intelligence](https://security-docs_2594.docs-preview.app.elstc.co/guide/en/security/master/es-ui-overview.html#_intelligence) - new section

### CODE REVIEW

**Review by nastasha-solomon — APPROVED** [review]
🚀🚀🚀

File: docs/getting-started/security-ui.asciidoc [hunk 1]
```diff
@@ -40,7 +40,7 @@ The {security-app} contains the following pages that enable analysts to view, an
 [float]
 === Dashboards
 
-Expand this section to access the Overview, Detection & Response, Kubernetes, and Cloud Posture dashboards, which provide interactive visualizations that summarize specific data within your environment. Refer to <<dashboards-overview>> for more information.
+Expand this section to access the Overview, Detection & Response, Kubernetes, Cloud Posture, and Entity Analytics dashboards, which provide interactive visualizations that summarize specific data within your environment. Refer to <<dashboards-overview>> for more information.
```
Comment (benironside) [inline docs/getting-started/security-ui.asciidoc:43]:
```suggestion
Expand this section to access the Overview, Detection & Response, Kubernetes, Cloud Posture, and Entity Analytics dashboards, which provide interactive visualizations that summarize your data. Refer to <<dashboards-overview>> for more information.
```

File: docs/getting-started/security-ui.asciidoc [hunk 2]
```diff
@@ -99,6 +99,19 @@ image::images/network-ui.png[Shows the Network page]
 [role="screenshot"]
 image::images/users/users-page.png[Shows the Users page]
 
+[float]
+=== Intelligence
+
+The Intelligence section contains the Indicators page, which collects data from enabled threat intelligence feeds and provides a centralized view of indicators of compromise (IoCs). Refer to <<investigate-events>> to learn more.
```
Comment (benironside) [inline docs/getting-started/security-ui.asciidoc:105]:
```suggestion
The Intelligence section contains the Indicators page, which displays data from enabled threat intelligence feeds and provides a centralized view of indicators of compromise (IoCs). Refer to <<investigate-events>> to learn more.
```
Collect seems to me like something that server agents do

**Review by benironside — COMMENTED** [review]
Couple of minor line edits, otherwise LGTM!

File: docs/getting-started/security-ui.asciidoc (hunk 2 above)
Comment (joepeeples) [reply docs/getting-started/security-ui.asciidoc:105]: @benironside I think it's trying convey that the page gathers data from disparate sources/feeds and presents them in a single place. "Displays" sounds like the data might have already been there, it just needed to be displayed.

@nastasha-solomon Any preference? This description was lifted from [your PR](https://github.com/elastic/security-docs/pull/2526) on the Indicators page.

Comment (nastasha-solomon) [reply docs/getting-started/security-ui.asciidoc:105]: Hm, I see your point @benironside but think I'd still prefer to keep "collects". The second part of the sentence already tells users that the page displays data ingested from enabled threat feeds.

**Pull request merged**